### PR TITLE
build: replace eslint with the biome for formatting and linting changes

### PR DESCRIPTION
**Pull request by zimeg**
### Type of change

- [x] New feature

### Summary

This PR replaces `eslint` with `biome` to match the changes in https://github.com/slack-samples/bolt-js-starter-template/pull/91 and resume the merges from the wonderful and kind @dependabot 🙏 

### Requirements

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "lint": "eslint --ext .js ."
+    "lint": "npx @biomejs/biome check *.js",
+    "lint:fix": "npx @biomejs/biome check --write *.js"
```
Comment (zimeg) [inline package.json:9]: This change is a similar change that's found in https://github.com/slack-samples/bolt-ts-starter-template/pull/131 - open to making changes here!

**Review by zimeg — COMMENTED** [review]
📝 Quick notes for the reviewers: No changes to logic are in this PR other than making the recommended changes from `biome` 🌲

**Review by seratch — APPROVED** [review]
Thanks for working on this migration! Other reviewers may have some suggestions on details but overall this PR looks good to me 👍

**Review by filmaj — APPROVED** [review]
YES! Love me some biome

**Comment by zimeg** [comment]
@seratch @filmaj And so the goodness of `biome` now spreads here 🙏 ✨

**Pull request merged**